default.nix

```
@@ -19,13 +19,18 @@ pog.pog {
       description = "The prefix to use in conjunction with system string when pinning paths";
       argument = "PREFIX";
     }
+    {
+      name = "derivations";
+      description = "The list of derivations to do whitelisted cachix push and pin";
+      argument = "DERIVATIONS";
+    }
```

Can we make this ... rigorous? I.e., if any of the names in the list are missing in omnix JSON (due to say developer typo), then the whole script should faily abruptly.

---

I had already tested that, the script doesn't fail abruptly, it just skips that typo.

---

By making it 'rigorous' I mean that the script should fail abruptly in those scenarios. Skipping (silently ignoring errors) means it is not rigorous.

---

Got it. I thought just the opposite.